airbyte-integrations/connectors/source-instagram/metadata.yaml

```diff
@@ -7,7 +7,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 6acf6b55-4f1e-4fca-944e-1a3caef8aba8
-  dockerImageTag: 3.0.10
+  dockerImageTag: 3.0.11
```

Comment:
@maxi297 @bazarnov what's your take on version bump when we're completely rewriting the connector? Should this be `4.0.0`? I get that it's compatible, buuuuut.

Reply:
I've run live tests on a few connections without a problem. I don't know if this implies that my changes wouldn't need to be breaking changes. The new thing is the Api parent Stream ([description](https://github.com/airbytehq/airbyte/pull/39504#issue-2353878097)), which passes the buz accounts to (now) child Streams. Anyway, I will hear experience call on this matter.

![image](https://github.com/airbytehq/airbyte/assets/168454423/316a4156-6517-4b7d-84fa-2e58782fedfe)

Reply:
If the state, config and record outputs all have the same format and it does not add relatively big features, I would consider this a patch

Reply:
Once there are no regressions in terms of a breaking change, there is no need to make a `major` update, but the `patch`. @aldogonzalez8 FYI